src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets

```diff
@@ -0,0 +1,89 @@
+<!--
+***********************************************************************************************
+Microsoft.Net.Sdk.Razor.Versions.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+  <!--
+      Initialize properties and items inferred using the project version. This file is not imported in projects referencing
+      MVC \ Razor 2.2 or earlier since values specified here are provided via targets and props imported from NuGet packages
+      such as Microsoft.AspNetCore.Razor.Design.
```

Comment:
Should `IsRazorCompilerReferenced` be here?

Reply:
Sure